app/controllers/concerns/chat.rb

```diff
@@ -3,8 +3,8 @@ module Chat
 
   included do
     helper_method def chat_opening_hours
-                    @chat_opening_hours ||=
-                      ChatOpeningHoursDecorator.decorate(Rails.application.config.chat_opening_hours)
+      @chat_opening_hours ||=
+        ChatOpeningHoursDecorator.decorate(Rails.application.config.chat_opening_hours)
                   end
```

Comment:
this is an awkward indentation! I can see why the above lines were indented so much. if you move this `end` left as well, will it complain?

Reply:
It was complaning actually:
```
app/controllers/concerns/chat.rb:8:19: W: Layout/DefEndAlignment: end at 8, 18 is not aligned with helper_method def at 5, 4.
                  end
```
I put the end on the right place and now we have 65 offences :D.